dashboard/app/models/script_level.rb

```diff
@@ -67,6 +67,36 @@ def oldest_active_level
     end
   end
 
+  def has_another_level_to_go_to?
+    if script.professional_learning_course?
+      !end_of_stage?
+    else
+      next_progression_level
+    end
+  end
+
+  def next_level_or_redirect_path_for_user user
+    level_to_follow = (level.unplugged? || stage.try(:unplugged?)) ? next_level : next_progression_level
+
+    if script.professional_learning_course?
+      if levels[0].try(:plc_evaluation?)
```

Comment:
So I think I still need a try here - plc_evaluation? is a method on evaluation multi and level may not be an evaluation multi